### DISCUSSION TITLE
How to use v-pagination slot

### DISCUSSION THREAD

**Discussion by imchanyo** [vuetifyjs/vuetify#19608] [Q&A]
slot 

- first
- last

slot of first and last is not working

- want 
<img width="795" alt="스크린샷 2024-04-15 오전 9 39 58" src="https://github.com/vuetifyjs/vuetify/assets/84080689/a62c3ec1-ed2f-4550-b628-5e9ac18ef1a1">

- current
<img width="562" alt="스크린샷 2024-04-15 오전 9 40 23" src="https://github.com/vuetifyjs/vuetify/assets/84080689/f4d9372f-c154-4961-8a39-aedbde2663e8">

I'd like to add another button at the far end, but isn't frist, last used for this?
I used <template #first> <template #last> and I can't see it on screen. How should I use it?

```js
  <v-pagination :length="5">
    <template #first>
      <div>sfsf</div>
    </template>

    <template #last>
      <div>sfsf</div>
    </template>
 </v-pagination>


```

**Comment by SonTT19** [comment]
You need to add `show-first-last-page` props to v-pagination

https://vuetifyjs.com/en/api/v-pagination/#props-show-first-last-page
